app/lib/messages.js

```diff
@@ -1,17 +1,17 @@
-function invalidLocationMessage(location) {
-  return `We can't find the location ${location.toLocaleUpperCase()}. Check the location is correct and try again.`;
+function invalidSearchMessage(search) {
+  return `We can't find the search ${search.toLocaleUpperCase()}. Check the search is correct and try again.`;
 }
 
-function emptyLocationMessage() {
-  return 'You must insert a location to find a surgery.';
+function emptySearchMessage() {
```

Comment:
There is only a single use of this function atm. It might be better to have the message in place and extract it when it is used several times. If it has been extracted and is important it might be worth testing it.